plugins/hls-semantic-tokens-plugin/test/testdata/T1.expected

```diff
@@ -0,0 +1,81 @@
+SemanticTokenOriginal {_tokenType = TTypeCon, _loc = Loc {_line = 6, _startChar = 18, _len = 3}, _name = "Set"}
+SemanticTokenOriginal {_tokenType = TFunction, _loc = Loc {_line = 6, _startChar = 23, _len = 6}, _name = "insert"}
```

Comment:
I think we could give these a `Pretty` instance and pretty print this stuff in a way that's easier to read. Something like:
```
6:23-6:29 TFunction "insert"
```
perhaps?

Reply:
compact the test output would look good surely